blockchain.go

```diff
@@ -1015,30 +1015,31 @@ func (b *Blockchain) commitBlock() (*flowgo.Block, error) {
 func (b *Blockchain) GetAccountStorage(address sdk.Address) (*AccountStorage, error) {
 	program := programs.NewEmptyPrograms()
 
-	env, err := fvm.NewTransactionEnvironment(
+	meterParameters := meter.DefaultParameters().
```

Comment:
you can skip this.  the default value is already max int64

Reply:
Nice. Simplified it in [37fac23](https://github.com/onflow/flow-emulator/pull/194/commits/37fac2343d3b38b3ff79032cd436b9f624c738a7)